pkg/ansible/paramconv/paramconv.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// Based on https://github.com/iancoleman/strcase
```

Comment:
Probably don't need the last line. We already have this license in the SDK
https://github.com/operator-framework/operator-sdk/blob/master/commands/operator-sdk/main.go#L1-L13